### ISSUE TITLE
EventHubsDiagnosticSource does not follow corefx DiagnosticSource recommendations

### ISSUE THREAD

**Issue by gambrose** [Azure/azure-event-hubs-dotnet#318]
`EventHubsDiagnosticSource` exposes events named like `Microsoft.Azure.EventHubs.Receive.Stop` which goes against [recommendations](https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/DiagnosticSourceUsersGuide.md#event-names) to not include the Listener name in the event name.

**Comment by serkantkaraca** [comment]
@TomMilos > Can you help on the comment?

**Comment by serkantkaraca** [comment]
Are you referring to this? 

_"DO - keep the names reasonably short (< 16 characters). Keep in mind that event names are already qualified by the Listener so the name only needs to be unique within a listener. Short names make IsEnabled() faster."_

**Comment by gambrose** [comment]
Yes I was, the event names should not be qualified with `Microsoft.Azure.EventHubs`.

I also think it may be worth considering splitting out `Receive` and `Send` as two distinct `DiagnosticListener`s as the recommendations say this should be considered as they are kind of two distinct use cases.

**Comment by serkantkaraca** [comment]
I will take a look once I have some time. Thx for pointing.
